libraries/AP_HAL_ChibiOS/hwdef/include/minimize_features.inc

```diff
@@ -38,4 +38,6 @@ define AP_MOTORS_FRAME_HEXA_ENABLED 1
 define AP_MOTORS_FRAME_OCTA_ENABLED 1
 
 define HAL_EFI_ENABLED 0
-define AP_ICENGINE_ENABLED 0
\ No newline at end of file
+define AP_ICENGINE_ENABLED 0
+
+define AP_CRSF_OUT_ENABLED 0
```

Comment:
This should be off by default on all vehicles, not just minimized vehicles (IMO).